wfl-slack-app/src/slack-api.js

```diff
@@ -0,0 +1,24 @@
+var fetch = require('isomorphic-fetch');
+var queryString = require('query-string');
+
+module.exports = {
+    getUserInfo: function(userId) {
+        var params = queryString.stringify({
+            token: process.env.SLACK_APP_TOKEN,
+            user: userId
+        });
+
+        return fetch(`https://slack.com/api/users.info?${ params }`)
+        .then(response => {
+            if (response.status === 200) {
+                return response.json();
+            }
+            else {
+                throw new Error('Cannot connect to the Slack API.');
+            }
+        })
+        .then(slackApiResponse => {
+            return slackApiResponse.user.real_name;
```

Comment:
Why not return the entire user object from this API?

Reply:
Good point. I can rework this to return the entire user object.

Reply:
Changed to return entire user object in latest code